modules/ROOT/pages/visibility.adoc

```diff
@@ -0,0 +1,122 @@
+
+API visibility rules use the Salesforce object *CommunityApi* to represent each API published in your community, and Salesforce https://help.salesforce.com/articleView?id=sf.user_groups.htm&type=5[Public Groups, role=external, window=blank] and https://help.salesforce.com/articleView?id=sf.security_about_sharing_rules.htm&type=5[Sharing Rules, role=external, window=blank] to provide a flexible permissions model.
+
+To set API visibility, link each public group of users to a sharing set, and grant each sharing set permission to see one or more APIs.
+
+== Visibility Setup
+
+Follow these steps to set up API visibility in your community.
+
+=== Configure Your Community for Visibility
+
+Follow these steps to configure your community for API visibility:
+
+. Create a new *ACM Community User Plus* profile based on the *Customer Community User Plus* profile:
+.. Open *Salesforce Setup* > *Profiles*.
+.. Select the *Customer Community User Plus* profile.
+.. Click *Clone*.
+.. Enter a name for the new profile that represents the use case, such as `External Developer`.
+.. Assign permissions that match the member user profile permissions described in the API Community Manager documentation section xref:create-community.adoc#configure-member-user-profile-permissions[Configure Member User Profile Permissions].
+. Configure your community to use the new *ACM Community User Plus* profile:
+.. Open *Setup* > *All Communities*.
+.. In the API Community Manager control panel, open *Community Administration* and open the *Members* tab.
+.. In the *Select Profiles* section, use the search dropdown to select *Customer*.
+.. Add the new *ACM Community User Plus* profile to the selected profiles.
+.. _Do not_ remove the *System Administrator* profile.
++
+If you remove the *System Administrator* profile, the system locks you out and removes all access to your community.
+.. Remove the existing *ACM Community User* profile.
+. Remove all profiles from your community sharing set:
+.. Open *Setup* > *Communities Settings*.
+.. In the *Sharing Sets* section, edit your community sharing set and remove all profiles from the *Selected Profiles*.
+.. If the *Selected Objects* in the sharing set include *CommunityApi*, remove *CommunityApi* from the *Selected Objects*.
+. If your community already has users, open *Setup* > *Users* and update each user to change the *User License* to *Customer Community Plus* and select the new *ACM Community User Plus* profile.
+. In the API Community Manager control panel, set the visibility of each of your APIs to *Members Only*.
+
+=== Define Member Groups (Public Groups)
+
+Use member groups, also known as public groups, to group members and then make individual APIs visible to the group. Visibility rules, also known as sharing settings, determine the APIs visible to a member group.
```

Comment:
Set one term as the primary in the text that introduces this topic set, and then use that one consistently.

Always avoid punctuation in topic titles.

Reply:
Use italics for also-known-as terms.

Reply:
determine the APIs visible to a member group > determine which APIs are visible....
Just a bit more obvious

Reply:
Fixed.